src/ui/hooks/comments/useAddCommentReply.tsx

```diff
@@ -77,7 +78,7 @@ export default function useAddCommentReply() {
 
         const newParentComment = {
           ...parentComment,
-          replies: [...parentComment.replies, newReply],
+          replies: [...parentComment.replies.filter((r: any) => r.id !== temporaryId), newReply],
```

Comment:
Same problem as the reply implementation above. The ID generated by the optimistic update data and the real server data are different, so this won't filter the optimistic data out.

Reply:
Same as above: this works in my experiments because Apollo doesn't work the way one would expect it to.